backend/README.md

```diff
@@ -31,7 +31,7 @@ $ poetry install
 You'll also need to clone a new `.env` file from the `.env.template` to store local configuration options. This is a one-time operation on first setup:
 
 ```bash
-$ cp .env.template .env  # (first time only)
+$ cp .env.template .env  # (first time only - or when environment variables have been updated)
```

Comment:
Personally I wouldn't want to run this when the variables have been updated since it will wipe out the versions I've actually modified (e.g. any secrets), but the point that we need to maintain this is a good one 😄